content/developers/api-reference/scitt-api/index.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "SCITT API (Preview)"
+description: "SCITT API Reference (Preview)"
+lead: "SCITT API Reference (Preview)"
+date: 2021-06-09T13:49:35+01:00
+lastmod: 2021-06-09T13:49:35+01:00
+draft: false
+images: []
+menu: 
+  developers:
+    parent: "api-reference"
+weight: 112
+toc: true
+aliases: 
+  - /docs/api-reference/scitt-api/
+---
+
+{{< note >}}
+The SCITT API is currently in preview and subject to change.
+{{< /note >}}
+
+The **S**upply **C**hain **I**ntegrity, **T**ransparency and **T**rust (SCITT) initiative is a set of [IETF standards](https://datatracker.ietf.org/group/scitt/documents/) for managing the compliance and auditability of goods and services across end-to-end supply chains.
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+DataTrails provides a SCITT implementation for developers looking to integrate Auditing and Compliance capabilities into their services.
+
+## Dependencies
+
+The following are required to complete this walkthrough:
+
+- [A DataTrails subscription](https://app.datatrails.ai/signup)
+- [DataTrails scripts](#data-trails-scripts)
+- [SBOM Tool](https://github.com/microsoft/sbom-tool)
+- [Sample Code](#sample-code)
+
+### Data Trails Scripts
+
+TODO: Add details for installing the datatrails scripts
+
+### Sample Code
+
+Create a directory and clone the [SCITT Examples](https://github.com/scitt-community/scitt-examples) repository
+
+```bash
+mkdir datatrails-scitt-demo
+cd datatrails-scitt-demo/
+git clone https://github.com/scitt-community/scitt-examples.git
+cd scitt-examples/nodejs/
+```
+
+## Sample
+
+The following will accomplish the following tasks:
+
+1. Create a signing key
+1. Register provenance for a new artifact
+1. Create and register an SBOM for the artifact
+1. Create an register attestation for the artifact
+1. Query DataTrails for the statements about the artifact
+
+## Create a Signing Key
+
+DataTrails supports multiple signing keys.
+(TODO: link to supported DataTrails Signing Keys\)<br>
+For a quick start, we'll create a jwk which DataTrails will cryptographically validate upon registration.
+
+1. Create a local signing key
+
+    ```shell
+    datatrails key create <parameters>
+    ```
+
+## Register Provenance for an Artifact
+
+When registering statements about an artifact, a common identifier is required to corelate a series of statements.
+A SCITT `CWT_Claims subject` property captures the identifier.
```

Comment:
Currently, while our implementation will accepts the cwt claims subject, we are still using feed to correlate a series of statements

Reply:
Gotcha, until we make the backend change, showing it duped in the script shows our current implementation with the directional change for the latest SCITT architecture.

Reply:
Yeh showing it duped is good, but the wording needs to correlate with the list events based on feed, otherwise the messaging gets confused.

We can't say we use `CWT_Claims subject` to correlate a series of statements here, then in later sections show that its actually something called `feed_id` when listing events?

Reply:
I've added an internal user story for tracking the updates to cwt_claims, released as an updated draft